js/plugins/iq.js

```diff
@@ -192,145 +203,111 @@ var IqPlugin= function (framework)
           eventParams = params[0];
           pcode = eventParams.params.pcode;
           playerId = eventParams.params.playerBrandingId;
-          eventMetadata = {};
-          eventMetadata.playerCoreVersion = eventParams.playerCoreVersion;
-          eventMetadata.pcode = pcode;
-          eventMetadata.params = eventParams.params;
-          eventMetadata.embedCode = eventParams.embedCode;
-          eventMetadata.playerUrl = eventParams.playerUrl;
-
-          if (this.ooyalaReporter)
-          {
-            this.ooyalaReporter._base.pcode = pcode;
-            OO.log("IQ: Reported: reportCustomEvent() for event: " + eventName + " with args:" + JSON.stringify(eventMetadata));
-            this.ooyalaReporter.reportCustomEvent(eventName, eventMetadata);
-            /* TODO: disable for now as this is already reported by reporter.js in core */
-            //this.ooyalaReporter.reportPlayerLoad();
-            //OO.log("IQ: Reported: reportPlayerLoad()");
-          }
-          else
-          {
-            OO.log("IQ: Tried reporting event: " + OO.Analytics.EVENTS.VIDEO_PLAYER_CREATED +
-                   " but ooyalaReporter is: " + this.ooyalaReporter);
-          }
+          eventMetadata = params[0];
+          eventMetadata.qosEventName = eventName;
+          this.ooyalaReporter._base.pcode = pcode;
+          this.ooyalaReporter.reportCustomEvent(eventName, eventMetadata);
```

Comment:
How come we are reporting video player created as a custom event as well?

Reply:
@aeng7 the custom event contains all of the parameters specified in the Analytics MVP doc (like core version, url, params, etc.). The analytics.js reportPlayerLoad doesn't sent any metadata

Reply:
Got it